packages/ap-audit/alliance_platform/audit/settings.py

```diff
@@ -0,0 +1,71 @@
+from typing import TypedDict
+
+from alliance_platform.base_settings import AlliancePlatformSettingsBase
+from django.db.models import Value
+from django.db.models.expressions import Func
+from django.db.models.functions import Concat
+
+
+class AlliancePlatformAuditSettingsType(TypedDict, total=False):
+    """Settings for the Audit package of the Alliance Platform
+
+    These can be set in the Django settings file under the ``ALLIANCE_PLATFORM`` key:
+
+    .. code-block:: python
+
+        ALLIANCE_PLATFORM = {
+            "AUDIT": {
+                # settings go here
+            }
+        }
+    """
+
+    #: The name of the action used to evaluate permissions for auditing a model. Defaults to 'audit'.
+    LIST_PERM_ACTION: str | None
+    #: The name of the permission used to evaluate whether auditing of any type can be performed. If it resolves to ``False``,
+    #: this will override individual audit permissions determined by ``LIST_PERM_ACTION``. Defaults to
+    #: ``"alliance_platform.audit.can_audit"``
+    GLOBAL_AUDIT_PERMISSION_NAME: str | None
+    #: The SQL function used to generate the display name for users in audit logs. Defaults to showing first name and last name
```

Comment:
could you include the actual code so it's more obvious what you can actually set here: `Concat("first_name", Value(" "), "last_name")`. Add a warning that if your user model doesn't include first_name + last_name fields you have to set this setting